### ISSUE TITLE
ZSH shell 

### ISSUE THREAD

**Issue by tamerlan99** [neoneggplant/EggShell#127]
Need new zhs shell for eggshell. Bash is not working on Catalina.

**Comment by sim0nov** [comment]
@tamerlan99 

Hi.

Our team - @EntySec - now developing new framework for exploiting iOS 10/11/12/13/14 beta. 
So it is only on the first stage of developing, but you can try it.

This will be the replacement for **EggShell**.
**EggShell** stopped working since iOS 13 released and we have a solution: 

* https://github.com/EntySec/cyclone

Also there is a new payload for iOS 13 developed by @enty8080:

* https://github.com/enty8080/ac1d

Thanks.

**Comment by 0xfffffff0075e2040** [comment]
Could you explain this further?

**Comment by enty8080** [comment]
@RPwnage 

Latest versions of macOS like Catalina and newer uses ZSH as default shell instead of bash. 

If you want to take some ideas, I am also developing some tools to attack iPhoneOS, macOS here: https://github.com/enty8080/membrane

And future of this will be presented at my HatSploit Framework here:
https://github.com/EntySec/HatSploit

**Comment by 0xfffffff0075e2040** [comment]
I was just asking because I'll try to make the neccessary migrations in my Community Fork of [EggShell](https://github.com/rpwnage/EggShell-Community-Fork)

**Comment by enty8080** [comment]
@RPwnage I know

**Comment by enty8080** [comment]
@RPwnage Instead of old esplios you can use my solution ==> https://github.com/enty8080/ac1d

**Comment by 0xfffffff0075e2040** [comment]
Is it open for contribution?

**Comment by enty8080** [comment]
@RPwnage Yes, you can use it and modify it. But do not forget to give original credits.
